### PR TITLE
feat: add a test server

### LINKED ISSUE #30: Implement simple test program for dir-index-html
Requesting a simple test program for purposes of being able to cross-browser test the dir-index-html files page itself. At present, it's only possible to browser-test using the flat html file with the short variable names themselves, which isn't terribly accurate when it comes to looking for display errors. Thank you!

### PR DESCRIPTION
**Pull request by Stebalien**
fixes #30

### CODE REVIEW

**Comment by Stebalien** [comment]
@jessicaschilling does this work for you?

File: test/main.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		Name: "short-film.mov",
+		Path: "short-film.mov",
+	}, {
+		Size: "1 KiB",
+		Name: "description.txt",
+		Path: "description.txt",
+	}},
+	Path:     "/ipfs/QmFoo",
+	BackLink: "/..",
+	Hash:     "QmFoo",
```
Comment (jessicaschilling) [inline test/main.go:39]:
```suggestion
	Hash:     "QmFooBarQXB2mzChmMeKY47C43LxUdg1NDJ5MWcKMKxDu7RgQm",
```
Real-length string here helps troubleshoot overflow errors.

File: test/main.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+var testData = listingTemplateData{
+	Listing: []directoryItem{{
+		Size: "25 MiB",
+		Name: "short-film.mov",
+		Path: "short-film.mov",
+	}, {
+		Size: "1 KiB",
+		Name: "description.txt",
```
Comment (jessicaschilling) [inline test/main.go:34]:
```suggestion
		Name: "this-piece-of-paper's-got-47-words-37-sentences-58-words-we-wanna-know.txt",
```
For troubleshooting long-name errors.

File: test/main.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		Path: "description.txt",
```
Comment (jessicaschilling) [inline test/main.go:35]:
```suggestion
		Path: "this-piece-of-paper's-got-47-words-37-sentences-58-words-we-wanna-know.txt",
```

**Pull request merged**